odo/backends/sql.py

```diff
@@ -692,6 +692,7 @@ def select_or_selectable_to_frame(el, bind=None, dshape=None, **kwargs):
             parse_dates=datetime_fields,
             dtype=other_dtypes,
             skip_blank_lines=False,
+            escapechar=kwargs.get('escapechar', '\\'),
```

Comment:
Had to find [the docs] (http://pandas.pydata.org/pandas-docs/version/0.18.1/generated/pandas.read_csv.html) to understand this better, but now...
```
escapechar : str (length 1), default None

    One-character string used to escape delimiter when quoting is QUOTE_NONE.
```
```
quoting : int or csv.QUOTE_* instance, default 0

    Control field quoting behavior per csv.QUOTE_* constants. Use one of QUOTE_MINIMAL (0), QUOTE_ALL (1), QUOTE_NONNUMERIC (2) or QUOTE_NONE (3).
```

Does this mean that the `escapechar` isn't used unless `QUOTE_NONE`, i.e. not the default and therefore not what's used here, is used?